app/src/main/java/org/fossasia/susi/ai/helper/DateTimeHelper.kt

```diff
@@ -23,7 +23,8 @@ object DateTimeHelper {
     val currentTime: String
         get() {
             val delegate = "hh:mm aaa"
-            return DateFormat.format(delegate, Calendar.getInstance().time) as String
+            val dateTime = DateFormat.format(delegate, Calendar.getInstance().time)
+            return if (dateTime is String) dateTime else ""
```

Comment:
What is the return type of format? If it is nullable string, then use datetime ?: ""

Or nullToEmpty